src/GithubFilesBundle/Service/GithubImport.php

```diff
@@ -165,10 +174,11 @@ public function saveAssociations($position, $content, $lsItemKeys, LsItem $lsIte
         $fieldsAndTypes = LsAssociation::allTypesForImportFromCSV();
         // We don't use is_child_of because that it alaready used to create parents relations before. :)
         // checking each association field
+        $sequenceNumber = $content[$position][trim($lsItemKeys['sequenceNumber'])];
         foreach ($fieldsAndTypes as $fieldName => $assocType){
             if (array_key_exists($fieldName, $lsItemKeys) && $cfAssociations = $content[$position][trim($lsItemKeys[$fieldName])]) {
                 foreach (explode(',', $cfAssociations) as $cfAssociation) {
-                    $this->addItemRelated($lsDoc, $lsItem, $cfAssociation, $frameworkToAssociate, $assocType);
+                    $this->addItemRelated($lsDoc, $lsItem, $cfAssociation, $frameworkToAssociate, $assocType, $sequenceNumber);
```

Comment:
Should all the associations be getting the `sequenceNumber` applied to them?  Or only the `isChildOf` ones?

Not that this looks like it will apply the **same** sequence number to all associations (exploded from `$cfAssociations`) which does not seem correct.

Reply:
well I don't know, I thought the sequence number should be applied to other items different of 'is Child of'. So are you saying that the sequence number should be imported just for the 'is Child Of' association type?

Reply:
For this, I think so.

Otherwise if you had a set of `isRelatedTo` (or other) associations separated by commas they would all get the **same** sequence number.  I would assume that if they needed sequence numbers the values would need to be different for each of the associations that are the same type of association (for the same origin) for them to be useful.